### ISSUE TITLE
Importing ajv@8.6.2/dist/standalone times out in Deno, fails on web

### ISSUE THREAD

**Issue by SeparateRecords** [postui/esm.sh#92]
It just started failing in CI today, seems to be after the most recent update.
Here's the error:

```
error: Import 'https://esm.sh/ajv@8.6.2/dist/standalone' failed: 408 Request Timeout
```

Same thing with `ajv@8.6.2/dist/jtd`.

Viewing from a web browser, this is the generated code:

```
/* esm.sh - error */
throw new Error("[esm.sh] " + "esbuild: Multiple exports with the same name \"default\"");
export default null;
```

Thanks for all your hard work on esm.sh!

**Comment by ije** [comment]
@SeparateRecords thanks, i will into it ASAP

**Comment by ije** [comment]
@SeparateRecords  fixed in e262204
